app/src/main/res/layout/activity_oneday.xml

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:orientation="vertical"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              tools:context="com.example.cristiana.weatherapp.OneDayActivity">
+
+    <ImageView
+        android:id="@+id/weatherImage"
```

Comment:
Currently, it's always raining in your app. Please add other weather pics and switch them based on the forecast. For example, consider downloading: http://www.alessioatzeni.com/meteocons/
Or maybe take your pick from: http://naldzgraphics.net/freebies/free-weather-icons/